FirstTask.py

```diff
@@ -0,0 +1,230 @@
+from os import system as sys
+
+class BankAccount:
+    def __init__(self, name, password, account_no, balance, account_type):
+        self.name = name
+        self.balance = balance
+        self.account_no = account_no
+        self.account_type = account_type
+        self.password = password
+
+    def account_number(self):
+        return self.account_no
+
+    def pass_code(self):
+        return self.password
+    
+    def check_balance(self):
+        return self.balance
+
+    def check_account_type(self):
+        return self.account_type
+    
+    def deposit_bill(self, bill):
+        if bill >= 0:
+            self.balance += bill
+            return True
+        else:
+            return False
+    
+    def draw_money(self, money):
+        if self.balance >= money:
+            self.balance -= money
+            return True
+        else:
+            return False
+    
+    def account_details(self):
+        print("Account # ", self.account_no)
+        print("Name: ", self.name)
+        print("Balance: ", self.balance)
+        print("Account Type: ", self.account_type)
+  
+class Bank:
+
+    ACCOUNT_TYPES = ["Current", "Saving", "Business"]
+    bank_accounts = []
+    logged_in_account = None
+
+    def total_account(self):
+        return len(self.bank_accounts)
+
+    def create_account(self, name, code, account_type):
+        account = BankAccount(name, code, 1000 + self.total_account() + 1, 0, account_type)
+        self.bank_accounts.append(account)
+        return account.account_no
+
+    def find_account(self, account_number):
+        for i in range(len(self.bank_accounts)):
+            if account_number == self.bank_accounts[i].account_number():
+                return True, self.bank_accounts[i]
+        return False, f"{account_number} Account Not Found!"
+
+    def login(self, account_number, code):
+        status, response = self.find_account(account_number)
+        if status:
+            if code == response.pass_code():
+                self.logged_in_account = response
+                return True, "Successfully Login!"
+        return False, f"{account_number} Account Not Found!"
+
+    def transfer_money(self, sender_account_number, receiver_account_number, amount):
+        status_1, response_1 = self.find_account(sender_account_number)
+        status_2, response_2 = self.find_account(receiver_account_number)
+        if status_1 and status_2:
+            if response_1.draw_money(amount):
+                response_2.deposit_bill(amount)
+                return True, "Money Successfully transferred!"
+        else:
+            if status_1 == False:

```

Comment:
We can just check it like this:
```suggestion
            if status_1:
```
Also, update similar checks below.